worker-policy/src/main/java/com/github/cafdataprocessing/worker/policy/ExecuteTaskData.java

```diff
@@ -692,7 +692,12 @@ private void setUpWorkerHandler(WorkerPolicyHandler workerPolicyHandler, String
         }
 
         workerPolicyHandler.setApplicationContext(corePolicyApplicationContext);
-        ((ClassifyDocumentApiDirectImpl) classifyDocumentApi).registerHandler(workerPolicyHandler);
+        try {
+            ((ClassifyDocumentApiDirectImpl) classifyDocumentApi).registerHandler(workerPolicyHandler);
+        } catch (final NullPointerException ex) {
+            logger.error("NullPointerException thrown while trying to register handler " + uniqueName + " for context: "
+                + corePolicyApplicationContext, ex);
+        }
 
         //Register policy
         PolicyType policyTypeToRegister = workerPolicyHandler.getPolicyType();
```

Comment:
You should probably throw or exit the method if that null pointer happens. Although the line above should have cleared it from being null....but then we wouldn't need the catch....